### ISSUE TITLE
 Could not find firebase-iid-interop.aar

### ISSUE THREAD

**Issue by hussainahmad** [invertase/react-native-firebase#1648]
**Versions:**
**react-native:0.57.0**
**react-native-firebase:5.0**
I want to implement FCM push notification in my react native Project, I follow the all steps as mentioned here
for [initial setup](https://rnfirebase.io/docs/v5.x.x/installation/initial-setup) when I am embedding android native code through this [android code](https://rnfirebase.io/docs/v5.x.x/messaging/android) but code is not running getting error. 
` What went wrong:
Could not resolve all files for configuration ':app:debugCompileClasspath'.
> Could not find firebase-iid-interop.aar (com.google.firebase:firebase-iid-interop:16.0.0).
  Searched in the following locations:
      https://jcenter.bintray.com/com/google/firebase/firebase-iid-interop/16.0.0/firebase-iid-interop-16.0.0.aar
`
Sorry  I am new in react native development

**Comment by w4ndry** [comment]
Have you added this config?

`allprojects {
    repositories {
        mavenLocal()
        google() // <-- Add this line above jcenter
        jcenter()
        maven {
            // All of React Native (JS, Obj-C sources, Android binaries) is installed from npm
            url "$rootDir/../node_modules/react-native/android"
        }
    }
}`

in `android/build.gradle`

**Comment by hussainahmad** [comment]
Thanks mate, Issue was resolved,  but after solving this issue facing this 
```
FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':app:dependencies'.
> The library com.google.android.gms:play-services-basement is being requested by various other libraries at [[10.2.4,10.2.4], [15.0.1,15.0.1]], but resolves to 15.0.1. Disable the plugin and check your dependencies tree using ./gradlew :app:dependencies.
```
Solved this issue by #1155

**Comment by lassikin** [comment]
I had to clear the gradle cache from homedir/.gradle/caches for it to start working again. The gradle cache had just frozen.
